presto-docs/src/main/sphinx/functions/comparison.rst

```diff
@@ -199,6 +199,10 @@ Examples::
     WHERE name LIKE '%#%%' ESCAPE '#'
     --returns 'a%c' and  '%cd'
 
+    SELECT * FROM (VALUES ('abc'), ('ab\nc') AS t (name)
+    WHERE name LIKE 'ab'
+    --returns 'ab'
```

Comment:
nit: shouldn't this returns nothing?

Reply:
thanks for catching this. I changed the example